app/assets/javascripts/application.js

```diff
@@ -77,4 +77,28 @@ $( document ).ready(function() {
       $(this).children(".plusminus").text('-');
     }
   });
-});
\ No newline at end of file
+});
+
+
+function hideNewsLetterValidation(email) {
+  if(email  != ""){
```

Comment:
Is it possible for email to be null?

Reply:
@daryanypl  Yes, this is from UI. Without entering if any one will click submit button, it should raise the validation.

Reply:
If it's possible to call this function with a null, I'm thinking we should check for null value as well as an empty string here.